src/monodroid/jni/inmemory-assemblies.h

```diff
@@ -0,0 +1,47 @@
+#ifndef INC_MONODROID_INMEMORY_ASSEMBLIES_H
+#define INC_MONODROID_INMEMORY_ASSEMBLIES_H
+
+#include "dylib-mono.h"
+#include "jni-wrappers.h"
+#include <string.h>
+
+#define DEFAULT_CAPACITY 8
+
+namespace xamarin { namespace android { namespace internal {
+	class InMemoryAssemblies
+	{
+		private:
+			struct InMemoryAssemblyEntry
```

Comment:
I added the mentioned .ctor  and .dtor

I'm 70% sure that because `InMemoryAssemblyEntry` is never manipulated other than by pointer I don't need the copy ctor and overloaded assignment operator but correct me if I'm wrong.